client/src/modules/Job/pages/NoAuthLanding/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Footer } from 'modules/Job/components/Footer';
+import { COLORS } from 'modules/Job/theme/colors';
+import { Header } from '../../components/Header';
+import { EmployerInfo } from './components/EmployerInfo';
+import { Info } from './components/Info';
+import { JobSeekerInfo } from './components/JobSeekerInfo';
+import { TopSlide } from './components/TopSlide';
+
+export function NoAuthLanding() {
+  return (
+    <>
+      <section className="w-1440">
+        <Header />
+        <TopSlide />
+        <div className="content-wrap">
+          <Info />
+          <JobSeekerInfo />
+          <EmployerInfo />
+        </div>
+        <Footer />
+      </section>
+
+      <style jsx>
+        {`
+          @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+          :global(*) {
+            font-family: 'Roboto';
```

Comment:
do we really need these fonts?

Reply:
this font is used in mockup https://www.figma.com/file/14QjlTRmxmzEOqsve6xsBr/RS_JOBS?node-id=412%3A13423&t=b7IjbmuUef7zyqJ3-0